datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/exception/DataHubDataFetcherExceptionHandler.java

```diff
@@ -21,7 +21,15 @@ public DataFetcherExceptionHandlerResult onException(DataFetcherExceptionHandler
     log.error("Failed to execute DataFetcher", exception);
 
     DataHubGraphQLErrorCode errorCode = DataHubGraphQLErrorCode.SERVER_ERROR;
-    String message = "An unknown error occurred.";
+    String message = exception.getMessage();
```

Comment:
I think the previous code that set message to "Unknown error" existed so that we wouldn't send internal exception messages back to the frontend. I think you should keep that behavior.